From 06520a2b42f242868260e389f7871a1b691e0525 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Sun, 4 Oct 2020 14:08:25 -0700
Subject: [PATCH 44/47] Tighten up scope wording
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Problem reported by Michael Deckers in:
https://mm.icann.org/pipermail/tz/2020-October/029299.html
* theory.html: Reword scope to be less ambitious.
* tz-link.html: Remove “computer-based”.
---
 theory.html  | 2 +-
 tz-link.html | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/theory.html b/theory.html
index 22b9815..4613799 100644
--- a/theory.html
+++ b/theory.html
@@ -33,7 +33,7 @@
 The <a
 href="https://www.iana.org/time-zones"><code><abbr>tz</abbr></code>
 database</a> attempts to record the history and predicted future of
-all computer-based clocks that track civil time.
+civil time scales.
 It organizes <a href="tz-link.html">time zone and daylight saving time
 data</a> by partitioning the world into <a
 href="https://en.wikipedia.org/wiki/List_of_tz_database_time_zones"><dfn>timezones</dfn></a>
diff --git a/tz-link.html b/tz-link.html
index b634082..606a83b 100644
--- a/tz-link.html
+++ b/tz-link.html
@@ -218,7 +218,7 @@ advance, as this will coordinate updates to many cell phones,
 computers, and other devices around the world.
 The change should be officially announced at least a year before it affects
 how clocks operate; otherwise, there is a good chance that some
-computer-based clocks will operate incorrectly after the change, due
+clocks will operate incorrectly after the change, due
 to delays in propagating updates to software and data. The shorter
 the notice, the more likely clock problems will arise; see "<a
 href="https://codeofmatt.com/2016/04/23/on-the-timing-of-time-zone-changes/">On
-- 
1.8.3.1

